### DISCUSSION TITLE
Can this be modified to work on regular websites that host movies/tv shows?

### DISCUSSION THREAD

**Discussion by FreddyBLtv** [pystardust/ani-cli#128] [Ideas] [answered]
This is such a simple but powerful tool! I'm curious to know if it could be modified to work for regular websites that host movies/tv shows. I would do it myself but I have no idea on how to go about it.

**Comment by geodesius** [comment]
Yeah, would love to see a movi-cli, maybe based on something like dopebox.to or moviesjoy.to. It's hard to find a show or movie that's not on these but I think they might have an annoying embed.

**Comment by Rendevior** [comment]
Such a good idea, Fine i'm gonna go do it. Probably may take 2 days ig. including solving errors

**Reply by FreddyBLtv** [reply]
Let's go! What's the update so far?

**Reply by Rendevior** [reply]
I'm done but haven't published yet, Probably Tomorrow.

**Reply by FreddyBLtv** [reply]
You are the goat! did you publish yet?

**Reply by aj-arts** [reply]
boy would I love to use that function you are a lifesaver @TheGlockMisc

**Reply by port19x** [reply]
tv-cli confirmed fork? Exciting stuff

**Comment by port19x** [comment, marked as answer]
https://github.com/Bugswriter/notflix
